tests/unit/scripts/editor/optionColors.test.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Unit tests for get-option-colors.js helper
+ *
+ * @group unit
+ */
+import { jest } from '@jest/globals';
+
+const { getOptionColors } = require("../../../../scripts/editor/get-option-colors");
+
+// Mock for getPaletteColors()
+const coreColors = {
+	"primary": {
+		"name": "Primary",
+		"slug": "primary",
+		"color": "#022687"
+	},
+	"black": {
+		"name": "Black",
+		"slug": "black",
+		"color": "#000000"
+	},
+	"white": {
+		"name": "White",
+		"slug": "white",
+		"color": "#FFFFFF"
+	},
+	"secondary": {
+		"name": "Secondary",
+		"slug": "secondary",
+		"color": "#05A8AA"
+	},
+	"caribbean": {
+		"name": "Caribbean",
+		"slug": "caribbean",
+		"color": "#06BDBF"
+	},
+	"transparent": {
+		"name": "Transparent",
+		"slug": "transparent",
+		"color": "transparent"
+	}
+};
+
+const mockGetPaletteColors = jest.fn().mockReturnValue(coreColors);
+
+jest.mock('../../../../scripts/editor/get-palette-colors', () => ({
+	__esModule: true,
+	getPaletteColors: mockGetPaletteColors
+}));
+
+
+it('tests optionColors helper returns correct color subset', () => {
+	const colors = getOptionColors(["primary", "white"]);
+
+	expect(colors).toBe(
+		[
+			{
+				"name": "Primary",
+				"slug": "primary",
+				"color": "#022687"
+			}, {
+				"name": "White",
+				"slug": "white",
+				"color": "#FFFFFF"
+			}
```

Comment:
```suggestion
			coreColors.primary,
			coreColors.white
```

No need to duplicate code.